browser/src/control/Control.Mention.ts

```diff
@@ -59,23 +59,34 @@ interface PopupData {
 	id: string;
 }
 
-L.Control.Mention = L.Control.extend({
-	onAdd: function(map: ReturnType<typeof L.map>) {
+interface MessageEvent {
+    typingMention: boolean;
+}
+
+class Mention {
+	map: ReturnType<typeof L.map>;
+	newPopupData: PopupData;
+	firstChar: string;
+	users: any;
+	itemList: Array<any>;
+	_map: ReturnType<typeof L.map>;
+	data: MessageEvent<any>;
+
+	onAdd (map: ReturnType<typeof L.map>) {
```

Comment:
We need `constructor()`, I guess we want to do in it what we did in `onAdd`.
Also please look where this component was used outside this file. Probably we do something like:
`map.addControl(mention)` - with regular class we don't want that (as it no longer uses Leaflet Class parent class).
`addControl` calls onAdd

Reply:
I didn' touch here now, because we still getting different errors. When they are solved it will be easier to change here